### ISSUE TITLE
🐛 [cloud_firestore] Using the plugin in a CLI or server app

### ISSUE THREAD

**Issue by KristianBalaj** [FirebaseExtended/flutterfire#6422]
Hey, I would like to use Cloud Firestore in a Dart CLI app. The result should be just a command line executable without any Flutter UI.

Is this possible to connect to the Cloud Firestore without the use of Flutter SDK? Do you have any tips?

Thanks in advance.

**Comment by markusaksli-nc** [comment]
Hi @KristianBalaj 
There currently isn't a pure-Dart Firebase SDK so the only way to use Firestore from Dart is the FlutterFire plugins, which wrap the native iOS, Android, and JS Firebase SDKs. Because these implementations rely on the SDK source code language runtime (Swift, Kotlin, JS) and the native setup they cannot be separated from the Flutter SDK.

You can look at using the unofficial Dart implementation of Firebase Auth and Firestore https://pub.dev/packages/firedart or try to customize the open-source plugin code in this repository to use one of the official SDKs through Dart interop.

TLDR: There is no simple way to setup and use Firebase from pure Dart
Thank you

**Comment by KristianBalaj** [comment]
Thanks for clarification 👍
